File: WindowsFormsApplication1/Engine.cs
using System;
using System.Collections.Generic;
using System.Data.SqlClient;
using System.Linq;
using System.Text;
using System.Data;

namespace WindowsFormsApplication1
{
    class Engine
    {
        public static string ErrorMsg = "";
        public static string cnstring="data source=d2k62\\sqlexpress;user id=sa;password=sa123;initial catalog=OVSepPOST";
        public static bool ExecuteQry(string qry)
        {
            ErrorMsg = "";
            SqlConnection cn = new SqlConnection(cnstring);
            SqlCommand cmd = new SqlCommand();
            cn.Open();
            cmd.Connection = cn;
            cmd.CommandText = qry;
            try
            {
                cmd.ExecuteNonQuery();
                cn.Close();
                return true;
            }
            catch (Exception ex)
            {
                cn.Close();
                ErrorMsg = ex.Message;
                return false;
            }
        }
        public static DataTable getDataTable(string sql,string tableName="Table1")
        {
            SqlConnection cn = new SqlConnection(cnstring);
            cn.Open();
            SqlDataAdapter ad = new SqlDataAdapter(sql, cn);
            DataTable dt = new DataTable(tableName);
            ad.Fill(dt);
            cn.Close();
            if (dt.Rows.Count > 0)
                return dt;
            else
                return null;
        }
    }
}

File: SidImport/Engine.vb

Imports System.Data.SqlClient

Class Engine
    Public Shared ErrorMsg As String = ""
    Public Shared cnstring As String = "data source=d2k62\sqlexpress;user id=sa;password=sa123;initial catalog=OVSepPOST"

    Public Shared Function ExecuteQry(ByVal qry As String) As Boolean
        ErrorMsg = ""
        Dim cn As SqlConnection = New SqlConnection(cnstring)
        Dim cmd As SqlCommand = New SqlCommand()
        cn.Open()
        cmd.Connection = cn
        cmd.CommandText = qry

        Try
            cmd.ExecuteNonQuery()
            cn.Close()
            Return True
        Catch ex As Exception
            cn.Close()
            ErrorMsg = ex.Message
            Return False
        End Try
    End Function

    Public Shared Function getDataTable(ByVal sql As String, Optional ByVal tableName As String = "Table1") As DataTable
        Dim cn As SqlConnection = New SqlConnection(cnstring)
        cn.Open()
        Dim ad As SqlDataAdapter = New SqlDataAdapter(sql, cn)
        Dim dt As DataTable = New DataTable(tableName)
        ad.Fill(dt)
        cn.Close()

        If dt.Rows.Count > 0 Then
            Return dt
        Else
            Return Nothing
        End If
    End Function
End Class

